### ISSUE TITLE
Is there a way to make new tabs?

### ISSUE THREAD

**Issue by Toadfield** [Swordfish90/cool-retro-term#611]
I like the look of this terminal emulator,but I like the ability to make new tabs like in Konsole.

**Comment by bsacharski** [comment]
@Toadfield why not use a terminal multiplexer like [tmux](https://github.com/tmux/tmux/wiki)?
That way your tabs would be independent from the terminal emulator.

**Comment by xslendix** [comment]
> @Toadfield why not use a terminal multiplexer like [tmux](https://github.com/tmux/tmux/wiki)?
> That way your tabs would be independent from the terminal emulator.

I agree with you here. Using sometging like tmux looks cooler and tabs would destroy the look

**Comment by allisonober** [comment]
So if you're looking for multiple terminal support, I found a way to have multiple windows at once (not tabs though.)

It's very limited, but simple to do. In MacOS, I copied the application and renamed it to make it a separate app. Now I have two windows to use, which act like separate applications.

<img width="1096" alt="Screen Shot 2020-10-16 at 10 35 33 AM" src="https://user-images.githubusercontent.com/52665928/96272375-0483e680-0f9c-11eb-9e49-9a7c4d130463.png">

The downsides are configs won't be shared (you'll have to export in one and import in the other to update your profiles), and you'll need to repeat this depending on how many extra windows you want.

The up side is this is a very easy technique that can likely work on any OS. And a bonus, you can cmd+tab between the terminals, which I actually prefer.

**Comment by alexmyczko** [comment]
you could use gnu screen?

**Comment by bufanda** [comment]
I would like to have tabs too. With tmux it's ok to work when it's just one project at a time. But when I work on multiple things I like to have multiple tabs open in iTerm or Terminator on Linux.
